Framework/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # This is mainly here so you don't get a complaint when running cmake
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.5)
```

Comment:
This should already be changed in [master](https://github.com/mantidproject/mantid/blob/master/Framework/CMakeLists.txt#L2)